cli/src/generate/templates/binding.cc

```diff
@@ -1,28 +1,16 @@
 #include "tree_sitter/parser.h"
-#include <node.h>
-#include "nan.h"
-
-using namespace v8;
+#include <napi.h>
 
 extern "C" TSLanguage * tree_sitter_PARSER_NAME();
 
 namespace {
```

Comment:
```suggestion
```

seems unnecessary and the examples don't have it https://github.com/nodejs/node-addon-examples/blob/main/src/1-getting-started/6_object_wrap/node-addon-api/addon.cc